src/SFA.DAS.EmployerApprenticeshipsService.Web/Content/Site.css

```diff
@@ -565,9 +565,9 @@ details .summary{
    padding: 4px;
 }
 
-/*remove focus from radio buttons*/
+/*remove focus from radio & checkboxes */
 
-input[type=radio]:focus{
+input[type=radio]:focus, input[type=checkbox]:focus{
     outline:0 !important;
```

Comment:
We shouldn't remove default focus checkboxes and radios, users need to be able to navigate with a keyboard